### DISCUSSION TITLE
Please don't drop .netstandard2 support for GrpcClient

### DISCUSSION THREAD

**Discussion by abe545** [open-telemetry/opentelemetry-dotnet#3617] [General]
~We still have full framework applications that make grpc calls - if you drop support for netstandard2.0 the GrpcClient library, then we can no longer instrument those calls. netstandard2.1 doesn't support any full framework version, so we'd have no path forward to using otel to instrument grpc (google is dropping support for the other client library, so we can't stick with it and use the contrib grpc client instrumentation).~
Something else must have been going on for me - I don't think you actually did this... Must have been the aspnetcore dependency update that I somehow misinterepreted

**Comment by alanwest** [comment]
We have no intention of dropping any .NET Framework support for the Grpc.Net.Client library.
